rpc/internal/cedar.ext.go

```diff
@@ -11,10 +11,11 @@ import (
 
 func ExportArtifactInfo(in *poplar.TestArtifact) *ArtifactInfo {
 	out := &ArtifactInfo{
-		Bucket: in.Bucket,
-		Prefix: in.Prefix,
-		Path:   in.Path,
-		Tags:   in.Tags,
+		Location: StorageLocation_CEDAR_S3,
```

Comment:
shouldn't this be in the input somewhere?

Reply:
I thought for this client we want all data to be in s3? I can add a Location field in the TestArtifact struct in report.go if we want to allow clients to chose where the data is uploaded